kolibri/plugins/facility_management/assets/src/views/user-page/index.vue

```diff
@@ -108,7 +78,7 @@
 
 <script>
 
-  import coreTable from 'kolibri.coreVue.components.coreTable';
+  import userTable from '../user-table';
```

Comment:
Could you fix the thing where Class Coaches are not being included when "Coaches" filter is selected?